bindings/nim/kzg_abi.nim

```diff
@@ -5,14 +5,14 @@
 import
   std/[strformat, strutils]
 
-from os import DirSep
+from os import DirSep, AltSep
 
 const
   # FIELD_ELEMENTS_PER_BLOB is overrideable from
   # compiler switch -d: or --define:
-  FIELD_ELEMENTS_PER_BLOB* {.intdefine.} = 4096
+  FIELD_ELEMENTS_PER_BLOB* {.strdefine.} = 4096
```

Comment:
This was just changed here: https://github.com/ethereum/c-kzg-4844/pull/294

Before merging this I think the rest of the nimbus team should review this to make sure that it's final.

Reply:
I'm a member of the Nimbus team, but this change was accidental. Will remove it from the PR.

Reply:
The fix has been pushed.